config/locales/es.yml

```diff
@@ -3103,20 +3104,20 @@ es:
           other: En %{year}, ¿usted o su cónyuge adoptaron un hijo?
       arp_payments:
         errors:
-          advance_child_tax_credit: Ingrese un valor para el crédito fiscal anticipado por hijos o díganos si no está seguro. Si no recibió CTC o no se aplica a usted, ingrese 0.
-          eip: Ingrese valores para cada pago de estímulo o díganos que no está seguro de cuánto recibió.
+          advance_child_tax_credit: Favor de ingresar el monto del pago por adelantado del Crédito Tributario por Hijos, o díganos que no está seguro. Si no recibió el CTC o no aplica a usted, ingrese el 0.
+          eip: 'Favor de ingresar los montos de cada pago de estímulo, o díganos que no está seguro cuánto recibió. '
         help_text_html:
-        - A partir de abril de 2020, se emitieron tres pagos de estímulo, anteriormente conocidos como Pagos de impacto económico, como parte del Plan de rescate estadounidense.
-        - Además, las familias con niños menores de 17 años han recibido pagos de CTC.
-        - "<strong>¡Por favor comparta su información de pago a continuación!</strong> Si no recibió un pago, ingrese 0."
+        - A partir de abril 2020, se emitieron tres pagos de estímulo, anteriormente conocidos como Pagos de impacto económico, como parte del Plan de Rescate Estadounidense.
+        - 'Además, se les han enviado los pagos de Crédito Tributario por Hijos o CTC a familias con hijos menores de 17 años de edad. '
```

Comment:
There is an unnecessary ` ` (space) character at the end of this string. It is not a problem, but it is unnecessary.